go/registry/api/sanity_check.go

```diff
@@ -268,9 +268,11 @@ func SanityCheckStake(
 		generatedEscrows[addr].StakeAccumulator.AddClaimUnchecked(StakeClaimForNode(node.ID), StakeThresholdsForNode(node, nodeRts))
 	}
 	for _, rt := range runtimes {
-		// Add runtime stake claims.
-		addr := staking.NewAddress(rt.EntityID)
-		generatedEscrows[addr].StakeAccumulator.AddClaimUnchecked(StakeClaimForRuntime(rt.ID), StakeThresholdsForRuntime(rt))
+		if rt.GovernanceModel == GovernanceEntity {
```

Comment:
For runtime governance the required stake should be in the runtime account. Is this checked correctly in all the places (registr runtime, committee scheduler, etc.)? I think some of these haven't been updated.

Reply:
Hmm, it's possible, I'll double-check.